ci-operator/config/openshift/hive/openshift-hive-mce-2.3.yaml

```diff
@@ -0,0 +1,212 @@
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  unit:
+    requests:
+      cpu: 1000m
+      memory: 4Gi
+tests:
+- as: unit
+  commands: make test
+  container:
+    from: src
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: verify
+  commands: make verify
+  container:
+    from: src
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+- as: coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  run_if_changed: \.go$
+  secret:
+    mount_path: /tmp/secret
+    name: hive-codecov-token
+- as: publish-coverage
+  commands: |
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
+    make coverage
+  container:
+    from: src
+  postsubmit: true
+  secret:
+    mount_path: /tmp/secret
+    name: hive-codecov-token
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.12"
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        # Creds for the `ci` user in the hive-team cluster
+        # TODO: Get a CI-owned account: DPTP-2348
+        export AWS_ACCESS_KEY_ID=$(cat /tmp/secret/hive-aws-creds/AWS_ACCESS_KEY_ID)
+        export AWS_SECRET_ACCESS_KEY=$(cat /tmp/secret/hive-aws-creds/AWS_SECRET_ACCESS_KEY)
+        # Override default location for pull secret
+        export PULL_SECRET_FILE=/tmp/secret/pull-secret/.dockerconfigjson
+        # Our AWS account is set up with this domain:
+        export BASE_DOMAIN=new-installer.openshift.com
+        # Configure a cluster name that our monitor/reaper can use to
+        # - discover clusters created by this test (prefix `hiveci-`)
+        # - know when they were created (embedded date stamp)
+        # Add the PR number for extra uniqueness.
+        # We'll exceed the 21c cluster name cutoff at PR 1Mi or some
+        # time in July of 4147, whichever comes first.
+        export CLUSTER_NAME=$(printf "hiveci-%x-%x" $(date +%s) $PULL_NUMBER)
+        CLOUD=aws make test-e2e
+      credentials:
+      - mount_path: /tmp/secret/hive-aws-creds
+        name: hive-aws-creds
+        namespace: test-credentials
+      - mount_path: /tmp/secret/pull-secret
+        name: ci-pull-credentials
+        namespace: test-credentials
+      dependencies:
+      - env: HIVE_IMAGE
+        name: hive
+      - env: RELEASE_IMAGE
+        name: release:latest
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: generic-claim
+- as: e2e-pool
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h30m0s
+    version: "4.12"
```

Comment:
...here, because we don't yet have clusterpools for 4.14 (or 4.13 for that matter).